Readme.md

```diff
@@ -85,6 +85,64 @@ echo "<a href='$url'>Login with LinkedIn</a>";
 
 ```
 
+You can easily implement your own session storage device and use it with this library. Just make sure your session
+storage class implements the `\Happyr\LinkedIn\Storage\DataStorageInterface` interface (or write an adapter that
+uses your class and implements that interface as shown below).
```

Comment:
Great that you documented this feature!
But you don't have to put the entire file in the readme, just make a link to the document.